works/serializers.py

```diff
@@ -0,0 +1,77 @@
+from rest_framework import serializers
+
+from . import models
+
+
+class WorkTypeSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = models.WorkType
+        fields = ('id', 'name', 'is_active')
+
+
+class ArtTypeSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = models.ArtType
+        fields = ('id', 'work_type', 'name', 'is_active')
+
+
+class IgualaSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = models.Iguala
+        fields = ('id', 'client', 'name', 'start_date', 'end_date', 'is_active')
+
+
+class ArtIgualaSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = models.ArtIguala
+        fields = ('id', 'iguala', 'art_type', 'quantity', 'is_active')
+
+
+class WorkSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = models.Work
+        fields = ('id',
+                  'executive',
+                  'contact',
+                  'current_status',
+                  'work_type',
+                  'iguala',
+                  'creation_date',
```

Comment:
I know that we are the only ones using this API, but the creation_date should not be modifiable.

http://www.django-rest-framework.org/api-guide/serializers/#specifying-which-fields-to-include